scripts/generate_climos

```diff
@@ -10,9 +11,14 @@ from dp.generate_climos import logger, generate_climos, dry_run_handler
 
 def main(args):
     if args.dry_run:
-        logger.info("DRY RUN")
         for filepath in args.filepaths:
-            dry_run_handler(filepath, args.climo)
+            filename = filepath.split("/")[-1][:-3]
+            outpath = (
+                os.path.join(args.outdir, filename + "_dry_run.txt")
+                if args.dry_output_to_file
+                else None
+            )
```

Comment:
I think we want to have the default behaviour log to the `python` `logging` module rather than to a file.

Reply:
I believe it defaults to `logging` module if `-f` argument is not provided because of the `if args.dry_output_to_file else None` statement.

Reply:
You are right it will do that. 

I feel like we can make this a bit more clear. Instead of generating the filename for the output we should have `None` as the default and allow the user to specify names if they so choose. This would mean turning `dry_output_to_file` into a `string` rather than a `boolean`. That way you could just pass that directly into the method without having to process it into `outpath`.

Reply:
Right, I actually implemented this way, because I thought it needs a static naming convention to be called in `wps_generate_climos`. But after double-checking, it is OK for the output file to have a user-defined name.

Reply:
@nikola-rados Ahh, there is another reason why I did this. Unlike `generate_prsn`, `generate_climos` can take multiple input files and produce multiple output files as well. Therefore, if a user gives one argument for `-f/--dry-file`, the output file will contain contents only for the last input file generated. For, example, if we run the following command;
```
generate_climos -o output ./tests/data/tiny_daily_pr.nc ./tests/data/tiny_daily_prsn.nc -p mean -n -f dry_run.txt
```
`dry_run.txt` will only have the dry_run info for `./tests/data/tiny_daily_prsn.nc`.

I have two ideas on this problem:
1. keep the static naming method that I used
2. let users give multiple output file names, but the number of them should match the same number of input arguments. Otherwise, throw an error.
```
generate_climos -o output ./tests/data/tiny_daily_pr.nc ./tests/data/tiny_daily_prsn.nc -p mean -n -f pr_dry_run.txt prsn_dry_run.txt
```